kimchi/src/circuits/polynomials/permutation.rs

```diff
@@ -257,32 +257,45 @@ impl<F: PrimeField, G: KimchiCurve<ScalarField = F>, OpeningProof: OpenProof<G>>
             // (w[1](x) + gamma + x * beta * shift[1]) * ...
             // (w[6](x) + gamma + x * beta * shift[6])
             // in evaluation form in d8
```

Comment:
Same comment than https://github.com/o1-labs/proof-systems/pull/2969/commits/8e13b285588c68c903cb8bb106f83c793bc9e1bc#r1965351096

Reply:
the link seems broken, not sure what you meant?

Reply:
Mmh, I don't remember :confused:. I guess it was while reviewing commit by commit, and the comment has not been added at the right place :confused: